webui/src/views/dashboard/components/torrentList.vue

```diff
@@ -1,52 +1,105 @@
 <template>
   <a-table
+    v-if="mdAndLarger"
     :columns="columns"
     :data="data?.data"
+    column-resizable
+    :bordered="{ cell: true }"
     :loading="!loading && !data"
-    :virtual-list-props="{ height: 350, threshold: 10 }"
```

Comment:
You must use either virtual list or pagination, otherwise the performance will be poor.